.github/ISSUE_TEMPLATE/user-story-template.md

```diff
@@ -25,6 +25,27 @@ Then
 - [ ] Checklist item
 - [ ] Meets the DOD
 
+**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)
+- [ ] User story is included
+- [ ] User role and type are identified
+- [ ] Acceptance criteria are included
+- [ ] Wireframes are included (if required)
+- [ ] Does this User Story need stakeholder approval?
+- [ ] Design / Solution is accepted by Product Owner
+- [ ] Dependencies are identified (technical, business, regulatory/policy)
+- [ ] Story has been estimated (under 13 spts)
+
+·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)
+- [ ] Passes developer unit and automated tests
+- [ ] UI meets accessibility requirements
+- [ ] Configuration changes are documented, documentation and designs are updated
+- [ ] Feature flagged if required
+- [ ] Passes code peer-review
+- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
+- [ ] Ticket is ready to be merged to main branch
+- [ ] Can be demoed in Sprint Review
+- [ ] Bugs or future work cards are identified and created
+- [ ] Reviewed and approved by Product Owner
```

Comment:
Same, we have "Approved by PO or Dev" pipeline in ZenHub, why do we need it here too?

Reply:
I believe the Idea was to create a checklist so that everyone can kind of visualize what all steps the user story has passed/finished. Ex: If all applicable points are ticked and ticket is moved to Pending PO column, PO can review and check the "Reviewed and approved by Product Owner" and move it to "Closed". 
Also let me know if we can change/update it a better way. I will modify accordingly.